Dockerfile

```diff
@@ -11,13 +11,13 @@ COPY ./package.json package.json
 COPY ./package-lock.json package-lock.json
 COPY ./prisma/schema.prisma prisma/schema.prisma
 
-RUN apt update && apt install -y \
-  g++ \
-  git \
-  make \
-  openssl \
-  python3 \
-  && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y --no-install-suggests \
+    g++ \
```

Comment:
Can you keep the indentation?

Reply:
fixed in https://github.com/ghostfolio/ghostfolio/pull/3614/commits/fd3823a70dcdb1c32e43478b955bbb2fc66ef6b3